### DISCUSSION TITLE
Github Student Pack granted, but denied Copilot Access

### DISCUSSION THREAD

**Discussion by andrefilipefmsilva** [community/community#126097] [GitHub Education] [answered]
### Hi everyone,

Hi,

I have gotten access to the Student Pack, as per e-mail received. However, connecting this account with Visual Studio Code and installing Github Copilot, I get the following error:

`2024-05-28 00:38:33.142 [info] [auth] Invalid copilot token: missing token: 403 

2024-05-28 00:38:33.143 [error] [auth] Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as Frozen105."`

![image](https://github.com/community/community/assets/123335350/7c1f5fad-d6ce-4ad3-ab4a-74d2abf5b141)

 What is going on? How can this be fixed? I could really use Copilot this time of the semester. 

Appreciate any help,

Frozen105

**Comment by playgude2** [comment]
repsonse #1

**Comment by andrefilipefmsilva** [comment, marked as answer]
Turns out I just had to click the "Signup for Github Copilot" button on the image and go through some steps. Now it's working.
